app/Filament/Subcontractor/Widgets/SubcontractorDashboardWidget.php

```
@@ -0,0 +1,111 @@
+            return [];
+        }
+
+        $vigilanceService = app(VigilanceService::class);
+        $compliance = $vigilanceService->scanCompliance($thirdParty);
+        $isCompliant = $compliance['compliant'];
+
+        $activeChantiersCount = $thirdParty->chantiers()
+            ->where('status', ChantierStatus::IN_PROGRESS)
+            ->count();
+
+        $pendingInvoices = SubcontractorSituation::where('subcontractor_id', $thirdParty->id)
```

PHPStan (2.2.7)

[error] Call to an undefined static method App\Models\Commerce\SubcontractorSituation::where().


(staticMethod.notFound)